### DISCUSSION TITLE
iOS Working Group

### DISCUSSION THREAD

**Discussion by ofTheo** [openframeworks/openFrameworks#7174] [General]
Tagging @dimitre @danoli3 as a general place to coordinate and plan iOS improvements. 
also @dorald @vaporstack @danbz showed interest in helping too

Main ToDo list: #6968 

Small stuff: 
- fix / standardize rotation issues #6076 #6053

Bigger stuff: 
- Get Apothecary building arm64 device libs and arm64 simulator libs separately #6966 
- Fix photo loading 
- Audio routing with bluetooth headphones #7163

If anyone else wants to join this group - just add a mention in the discussion.

**Comment by dimitre** [comment]
I'll link here a personal list https://github.com/openframeworks/openFrameworks/issues/6968

**Comment by petewerner** [comment]
Hi I can help out with testing

**Comment by ofTheo** [comment]
adding @dorald 🙂

**Reply by dorald** [reply]
Thank you.

**Comment by ofTheo** [comment]
adding @vaporstack and @danbz

**Comment by vaporstack** [comment]
I'm sorry but I can't actually find the nightly builds on the oF website? I'm sure there's a reason I should be working against nightly instead of git source tree but idk what that reason is?   Cheers~

**Comment by ofTheo** [comment]
Apologies! They are at the bottom of the downloads page: https://openframeworks.cc/download/ 

It can be a bit more work to get projects running with the git source tree as you have to pull in the sub modules, download the third party libs and then run the project generator to make the example files. 

The nightly is basically all of those steps skipped 😁

**Comment by ofTheo** [comment]
Adding #7291 to this group discussion.

**Comment by ofTheo** [comment]
@artificiel would you be interested in helping with iOS related issues? 
Its probably the area of OF that needs the most love right now and is probably the easiest to contribute to, being that a lot of the functionality is in the addon and not the core.
